tools/run_tests/jobset.py

```diff
@@ -210,7 +210,7 @@ def state(self, update_cache):
       # TODO: looks like jenkins master is slow because parsing the junit results XMLs is not
```

Comment:
how about we update the comment to say that it's not the main cause of slowness but we still want to make sure we don't store overly huge logs.

Btw, do you think that an average stacktrace would be about 2KB in size? My very rough guess would be that it's a bit bigger. Let's try 4-10KB?